lib/grntest/worker.rb

```diff
@@ -181,6 +185,7 @@ def run_test(test_script_path, test_name)
 
           runner = TestRunner.new(@tester, self)
           return true if runner.run
+          return false if @result.n_leaked_tests > 0
```

Comment:
Why do we need this?
It seems that `TestRunner#run` returns `false` when leaked test exists: https://github.com/groonga/grntest/blob/master/lib/grntest/test-runner.rb#L111